controller/error_pages.rst

```diff
@@ -336,3 +336,49 @@ time and again, you can have just one (or several) listeners deal with them.
     your application (like :class:`Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException`)
     and takes measures like redirecting the user to the login page, logging them
     out and other things.
+
+Dumping error pages in static HTML files
+----------------------------------------
+
+When a web server cannot handle a request or when it triggers an error without
+calling the PHP application, it will return its default error pages, instead of
+rendering the errors as defined in your application (whether it's Symfony's
+default "Oops" error page or the pages you customized in your application).
+
+To avoid that and always have your web server rendering your application's error
+pages, you can dump each HTTP status error in a their own static HTML files:
+
+.. code-block:: terminal
+
+    $ php bin/console error:dump [--force] var/cache/prod/error_pages
+
+.. note::
+
+    By default, it will dump HTML files for each HTTP status error.
+    You can restrict that to dump only some HTTP status code by passing them as
+    in the second argument of the command.
+
+Once the static pages are generated, you can now configure your web server to use
+them instead of their default error page. Here is an example configuration with
+nginx:
```

Comment:
wdyt adding a doc for [frankenphp](https://caddyserver.com/docs/caddyfile/directives/handle_errors#examples) ?

Reply:
I don't use Frankenphp, so feel free to suggest a working configuration for it (or for any other web server) 😉

Reply:
We can add new config formats in future PRs if folks contribute them. Thanks.

Reply:
i will try to when I upgrade to v7.3, bookmarking this comments flows